runtime/compiler/net/LoadSSLLibs.cpp

```diff
@@ -448,6 +462,15 @@ bool loadLibsslAndFindSymbols()
    OX509_STORE_add_crl = (OX509_STORE_add_crl_t *)findLibsslSymbol(handle, "X509_STORE_add_crl");
    OX509_free = (OX509_free_t *)findLibsslSymbol(handle, "X509_free");
 
+   OEVP_MD_CTX_new = (OEVP_MD_CTX_new_t *)findLibsslSymbol(handle, (ossl_ver == 0) ? "EVP_MD_CTX_create"
+                                                                                   : "EVP_MD_CTX_new");
+   OEVP_MD_CTX_free = (OEVP_MD_CTX_free_t *)findLibsslSymbol(handle, (ossl_ver == 0) ? "EVP_MD_CTX_destroy"
+                                                                                     : "EVP_MD_CTX_free");
+   OEVP_DigestInit_ex = (OEVP_DigestInit_ex_t *)findLibsslSymbol(handle, "EVP_DigestInit_ex");
+   OEVP_DigestUpdate = (OEVP_DigestUpdate_t *)findLibsslSymbol(handle, "EVP_DigestUpdate");
+   OEVP_DigestFinal_ex = (OEVP_DigestFinal_ex_t *)findLibsslSymbol(handle, "EVP_DigestFinal_ex");
+   OEVP_sha512_256 = (OEVP_sha512_256_t *)findLibsslSymbol(handle, "EVP_sha512_256");
```

Comment:
The API in general - yes, but it turns out that 1.0.x doesn't have `EVP_sha512_256`. I will change the code to use `EVP_sha256` with OpenSSL 1.0.x. I chose SHA-512/256 because it's normally faster than SHA-256 on 64-bit platforms, and doesn't use the whole 512 bits (and was also faster than SHA-512 in my measurements).

Reply:
Done. The implementation now uses SHA-256.